src/functions/process.go

```diff
@@ -78,10 +80,18 @@ func processKeyValues(keyValues []string) {
 
 	for _, pair := range keyValues {
 		key, value := parseKeyValuePair(pair)
-		if key == "" {
+		if key == constants.EmptyAsString {
```

Comment:
```suggestion
		if key == constants.EmptyAsString {
```
No entendí la necesidad de este cambio, dentro de constants, EmptyAsString es literalmente `""`, exactamente lo mismo que se tenía en la versión anterior... Sólo se invoca desde este lugar, va ser reutilizado por N lugares? 
Sería necesario cuando (valga la redundancia) se vea la necesidad de utilizar la util por todos lados, no ahora que solo es utilizado en `process.go`

Reply:
Si esta medio alpedo solo me dio tock por que no tenia nombre, a lo sumo tendria que ir como variable en el mismo archivo

Reply:
Por ahora lo dejo en el mismo archivo (me sigue dando tock que no tenga nombre)